docs/application/vscode-ext/web.md

```diff
@@ -240,11 +328,62 @@ To run project:
 To debug application, follow these steps:
 
 1. In VS Code, click **View** > **Command Palette**.
-2. In the input field of the **Command Palette** that appears, type **Run Web Simulator** and press **Enter**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Web Simulator** and press **Enter**.
 3. After the **Web Simulator** is launched, enter the path of the file in the box. For example, file:///C://Workspace/basic/index.html.
 
     ![Debugging you application in the Web Simulator](media/tizen_web18.png)
 
+### Run Design Editor
+
+To run design editor, follow these steps:
+
+> **Note** 
+> 
+> **You must select the HTML extension file in the file explorer.**
+
+To run Design Editor, follow these steps:
+
+You can launch Design Editor in one of the following ways:
+- Command Palette
+- Design-Editor button
+- Right-click preference
+
+**Command Palette**
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+
+    A new tab is created and the Design Editor is launched.
+
+**Design-Editor button**
+1. To launch the Design Editor, click Design-Editor at the bottom right of the screen.
```

Comment:
As there is only one list item here, how about writing as a just simple sentence, not in list form?
```suggestion
To launch the Design Editor, click **Design-Editor** at the bottom right of the screen.
```

Reply:
done.